eng/packages/http-client-csharp/generator/TestProjects/Local/Basic-TypeSpec/src/Generated/BasicTypeSpecModelFactory.cs

```diff
@@ -145,15 +145,15 @@ public static Friend Friend(string name = default)
         public static ProjectedModel ProjectedModel(string name = default)
         {
 
-            return new ProjectedModel(name, null);
+            return new ProjectedModel(name, additionalBinaryDataProperties: null);
         }
 
         /// <summary> The ReturnsAnonymousModelResponse. </summary>
         /// <returns> A new <see cref="Models.ReturnsAnonymousModelResponse"/> instance for mocking. </returns>
         public static ReturnsAnonymousModelResponse ReturnsAnonymousModelResponse()
         {
 
-            return new ReturnsAnonymousModelResponse(null);
+            return new ReturnsAnonymousModelResponse(additionalBinaryDataProperties: null);
```

Comment:
well this is different. Do we have a plan of changing this in the generator or normalize it in all generated libraries?

Reply:
This is caused by the version bump in MGC.
Check the latest generated code in MGC:
https://github.com/microsoft/typespec/blob/baff7608b58e0c4f45abe42b0f2d53ad3d2d0f00/packages/http-client-csharp/generator/TestProjects/Local/Unbranded-TypeSpec/src/Generated/UnbrandedTypeSpecModelFactory.cs#L127